handlers/validator.go

```diff
@@ -1991,12 +1991,21 @@ func ValidatorSync(w http.ResponseWriter, r *http.Request) {
 			startEpoch = firstShownEpoch - limit
 			endEpoch = firstShownEpoch
 		}
-		syncDuties, err := db.BigtableClient.GetValidatorSyncDutiesHistoryOrdered(validatorIndex, startEpoch, endEpoch, ascOrdering)
+		syncDuties, err := db.BigtableClient.GetValidatorSyncDutiesHistoryOrdered([]uint64{}, startEpoch, endEpoch, ascOrdering)
```

Comment:
I checked how much longer a query for all validators instead of a single validator takes and it is about 1.5 times the time.
This still seemed reasonable to me.